react-function-components/src/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const div = document.querySelector('#root');
+function CustomButton() {
+  return (
+    <button>Click Me!</button>
+  );
+}
+
+ReactDOM.render(CustomButton(), div);
```

Comment:
Don't call your components before passing them to the render method, let the render method choose when to call them.